numba_cuda/numba/cuda/cudadrv/_info.py

```diff
@@ -0,0 +1,19 @@
+# Maps CTK version -> (min supported cc, max supported cc) inclusive
+_CUDA_CC_MIN_MAX_SUPPORT = {
+    (11, 1): ((3, 5), (8, 0)),
+    (11, 2): ((3, 5), (8, 6)),
+    (11, 3): ((3, 5), (8, 6)),
+    (11, 4): ((3, 5), (8, 7)),
```

Comment:
Wikipedia says that 8.7 wasn't added until CUDA 11.5 but I cannot find a definitive reference in the CUDA docs for this. 😕